core/web/jsonld/Organization.php

```diff
@@ -0,0 +1,377 @@
+<?php
+
+namespace luya\web\jsonld;
+
+/**
+ * JsonLd - Organization
+ *
+ * @see http://schema.org/Organization
+ *
+ * @author Alex Schmid
+ * @since 1.0.0
+ */
+class Organization extends Thing implements OrganizationInterface
+{
+    use OrganizationTrait;
+
+    /**
```

Comment:
Could you please move the private vars into the OrganizationTrait?

Reply:
Sure :)